node/src/components/consensus/era_supervisor.rs

```diff
@@ -422,6 +442,7 @@ where
         rng: &'a mut NodeRng,
     ) -> Effects<Event<I>> {
         let current_era = self.current_era;
+        info!(?current_era, "current era");
```

Comment:
I think we can remove it now after we've done with debugging.

Reply:
I'll change it to `trace` - it can still prove useful in the future.